example-policies/example-policy2.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: delivery.knative.dev/v1alpha1
+kind: Policy
+metadata:
+  namespace: default
+  name: example-policy2
+spec:
+  mode: time
+  defaultThreshold: 60
+  stages:
+  - percent: 1
+  - percent: 10
+  - percent: 100
```

Comment:
is 100 needed? Should there be a validation that no stages are 0 or 100?

Reply:
sorry this was a typo (I shouldn't have written 100). The validation would have rejected this value.